psl/psl/tests/attributes/default_negative.rs

```diff
@@ -512,6 +512,69 @@ fn dbgenerated_default_errors_must_not_cascade_into_other_errors() {
     expectation.assert_eq(&error)
 }
 
+#[test]
+fn must_error_on_too_small_argument_in_nanoid() {
```

Comment:
It's not super obvious, but we've been slowly moving these validation tests to a new framework in `psl/psl/tests/validation`.

These work so that you create a new directory there, e.g. `nanoid`, then add `.prisma` files for every test you want to add. For example, this test would have the following content:

```prisma
datasource db {
  provider = "mysql"
  url = "does_not_matter"
}

model Category {
  id String @id @default(nanoid(1))
}
```

You then execute the tests. If the directory is `nanoid`, run `cargo test validation::nanoid` and you see this new test failing. Run it again `env UPDATE_EXPECT=1 cargo test validation::nanoid::test_name` and it will add the expected error to the end as a comment.

If there is no comment after the PSL definition, the data model does not give any validation errors.

Reply:
Awesome, thanks for the info! That new framework sounds great. I will go ahead and convert these tests over.